### ISSUE TITLE
Libpcre2-8 package issues on Rocky Linux 9

### ISSUE THREAD

**Issue by jshwhitlow** [remicollet/remirepo#235]
If you want to read what lead me to believe this was an issue with a package, you can read my full explanation [here](https://stackoverflow.com/questions/76277504/php-8-1-rocky-linux-9-preg-replace-compilation-failed-unrecognised-compi).

It was suggested that I raise an issue with the package maintainer.

I have the same issue as listed here (except I'm running PHP 8.1.8, not 7.4):
https://bugs.php.net/bug.php?id=81640

The solve was to update the Libpcre2-8 package, however, there are no available updates for me to install against that package from the Remi Repo. I am on version 10.40 for the pcre2 package. I installed all available updates for all packages as of today. 

I installed from this repo: http://rpms.remirepo.net/enterprise/remi-release-9.rpm

**Comment by jshwhitlow** [comment]
Well, seemingly an extremely recent patch to one of the packages this week fixed it. I ran an update just now and had several updates to install. The package in question wasn't one of them, but one of them fixed the problem.

Not sure which package, but it was one of these:
<img width="2466" alt="image" src="https://github.com/remicollet/remirepo/assets/17415608/0ae71e4d-25f7-4e6d-acab-cb0b27acff79">

**Comment by remicollet** [comment]
Yes known issue when PHP is built using 10.40 (from EL-9.1) and run with 10.37 (in EL-9.1)

Full system update solves this.

For memory
```
$ rpm -qp --requires php-common-8.1.19-1.el9.remi.x86_64.rpm 
...
pcre2(x86-64) >= 10.40
...
```

So this dependency ensure proper version is used.
So only outdated version are affected (and such config cannot be supported)
